tests/python/frontend/tflite/test_forward.py

```diff
@@ -472,6 +472,55 @@ def test_forward_convolution():
     _test_convolution([4, 17, 17, 12], [3, 3, 12, 2], [1, 1], [2, 2], 'VALID', 'NHWC', True)
 
 
+#######################################################################
+# Transpose Convolution
+# -----------
```

Comment:
Keep the same length of Transpose convolution

Reply:
fixed